firefox-ios/Client/Frontend/Strings.swift

```diff
@@ -3700,6 +3700,156 @@ extension String {
         comment: "Question shown to user when tapping an SMS or MailTo link that opens the external app for those.")
 }
 
+// MARK: Enhanced Tracking Protection/Unified Trust Panel
+extension String {
+    public struct EnhancedTrackingProtection {
+        public static let onTitle = MZLocalizedString(
+            key: "Menu.EnhancedTrackingProtection.On.Title.v128",
```

Comment:
nit: missed this on the first review, but do we want to wrap this in a `Menu` struct for organization purposes

Reply:
yes, that's a good idea